test/balancer_test.go

```diff
@@ -346,3 +352,153 @@ func (s) TestNonGRPCLBBalancerGetsNoGRPCLBAddress(t *testing.T) {
 		t.Fatalf("With both backend and grpclb addresses, balancer got addresses %v, want %v", got, nonGRPCLBAddresses)
 	}
 }
+
+const attrBalancerName = "attribute-balancer"
+
+// attrBalancerBuilder builds a balancer and passes the attribute key and value
```

Comment:
Do you think you would be able to use the stub balancer instead:
https://github.com/grpc/grpc-go/pull/3431/files#diff-ef2472da4a075e04e3580d038b3e14aa

If so, please use it (and add it to this PR, since the other PR will take another week or so to submit).

Reply:
Done. 

I had to embed the `balancer.Balancer` interface though to get it to compile. But I added a TODO there to remove it once the above mentioned PR goes in.

Reply:
Cool, I'm glad it worked out, thanks.